### PR TITLE
defect #1649017: upgrades and compatibility problems

### PR DESCRIPTION
**Pull request by pikachugb**
covered in PR:
- upgraded intellij plugin version to 1.5.3
- upgraded gradle to 7.4.2
- updated build.gradle to be compatible with the new versions
- solved some compatibility problems
- removed some unused pieces of code that were causing deprecation and console errors/warnings

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -42,7 +45,7 @@ sourceSets {
 }
 
 dependencies {
-    compile("com.hpe.adm.octane.ideplugins:octane-plugin-common:$commonVersion"){
+    implementation("com.hpe.adm.octane.ideplugins:octane-plugin-common:$commonVersion") {
```
Comment (alexandra-marinescu-microfocus) [inline build.gradle:48]: why change the dependency from compile to impl?

File: build.gradle [hunk 2]
```diff
@@ -64,20 +67,20 @@ dependencies {
...
         exclude group: 'com.google.guava', module: 'guava'
     }
 
-    testCompile group: 'junit', name: 'junit', version: '4.11'
+    testImplementation group: 'junit', name: 'junit', version: '4.11'
 }
 
 intellij {
-    plugins = ['java']
+    plugins = ['com.intellij.java']
```
Comment (alexandra-marinescu-microfocus) [inline build.gradle:83]: do we want to run with the java from inside intelliJ?can't this be an issue for users who have older versions of IntelliJ?

File: gradle.properties [hunk 3]
```diff
@@ -16,9 +16,9 @@ projectGroup = com.hpe.adm.octane.ideplugins
 projectVersion = 2020.2-16.0.400.1
 
 #ide plugin common version, configurable from outside for CI
-commonVersion = 1.7.1
+commonVersion = 1.7.2-SNAPSHOT
```
Comment (alexandra-marinescu-microfocus) [inline gradle.properties:19]: is this the common version with java 11?

Comment (pikachugb) [reply gradle.properties:19]: yes

File: build.gradle (hunk 2 above)
Comment (pikachugb) [reply build.gradle:83]: I'll discuss this with you

**Review by vladbenchea — APPROVED** [review]

**Review by alexandra-marinescu-microfocus — APPROVED** [review]

**Comment by atothhpe** [comment]
👏

**Pull request merged**